src/javaRosa.js

```diff
@@ -1117,9 +1117,7 @@ define([
 
         var items = getItextItemsFromMugs(form, true);
         items = _.object(_.map(items, function (item) { return [item.id, item]; }));
-        cells = nextRow();
-        while (cells) {
-            // what's the point of creating items here?
+        for (cells = nextRow(); cells; cells = nextRow()) {
```

Comment:
Unless I'm missing something I think you're using the feature. The statement is an assignment, which in python would evaluate to nothing, but in javascript evaluates to the right side of the assignment. Therefore I think both constructs rely on that feature.